test/test_ops.py

```diff
@@ -1357,8 +1357,12 @@ def test_slice_fancy_indexing_tuple_with_tensors(self):
     helper_test_op([(2,5,6,5,3,4)], lambda x: x[(a,(1,1))], lambda x: x[(i,(1,1))])
     helper_test_op([(2,5,6,5,3,4)], lambda x: x[(a,b,c,d,e)], lambda x: x[(i,j,k,o,p)])
 
-  def test_slice_fancy_indexing_errors(self): ...
-    # TODO: currently we not support IndexError for out of bounds idx values
+  def test_slice_fancy_indexing_errors(self):
+    a = Tensor.ones(10,11,12)
+    with self.assertRaises(IndexError): a[Tensor(1.1)] # tensors used as indices must be int or bool tensors
```

Comment:
nit: i find putting test case comments in a separate line above the code more readable. also fits the same stype as other tests in this file. fine to change in the next pr